util/runtests.sh

```diff
@@ -144,21 +144,31 @@ run_unittests python3
 TESTLOG=./integtest-full-$DATE.log
 # make sure it is full path
 TESTLOG="$(readlink -m "$TESTLOG")"
-INTEGLOGDIR="./integtest-$DATE/"
-mkdir -p "$INTEGLOGDIR"
+INTEGLOGDIR="./integtest-$DATE"
+mkdir -p "$INTEGLOGDIR-python2/"
+mkdir -p "$INTEGLOGDIR-python3/"
 
 # only echo ERRORs in the output
 touch "$TESTLOG" # To make sure tail doesn't fail
 tail -f "$TESTLOG" | grep --line-buffered "ERROR:" &
```

Comment:
Surprisingly, `tail -f` does quite different thing from `tail`. It opens the file, and keep checking if there is more line. Every new line is output. It never exits (unless you kill it).
So it's used for the terminal output while the tests are running.